lib/krikri/mapper.rb

```diff
@@ -135,7 +135,7 @@ def run(activity_uri = nil)
             rec.mint_id! if rec.node?
             activity_uri ? rec.save_with_provenance(activity_uri) : rec.save
           rescue => e
-            Rails.logger.error("Error saving record: #{rec.rdf_subject}\n" \
+            Rails.logger.error("Error saving record: #{rec.try(:rdf_subject)}\n" \
```

Comment:
That's interesting.  I'm not familiar with `try` as a method.  Can you tell me where it's defined, and what classes inherit it?  (Where could I expect to be able to use it?)